package.json

```diff
@@ -56,6 +56,7 @@
     "generate-types:settings": "./scripts/graphql-cli.sh codegen --project settings",
     "generate-types:proxy": "./scripts/graphql-cli.sh codegen --project proxy",
     "generate-types": "yarn generate-types:mc && yarn generate-types:ctp && yarn generate-types:settings && yarn generate-types:proxy",
+    "canary:uikit": "manypkg upgrade @commercetools-uikit canary",
```

Comment:
Thanks but I personally wouldn't add this script. You can always do it like `yarn versions:uikit canary`.

If you want, you can mention this in the README or something.